tests/lib/rules/camelcase.js

```diff
@@ -790,6 +794,25 @@ ruleTester.run("camelcase", rule, {
                 }
             ]
         },
+        {
+            code: "const {mem_var = 1} = obh",
```

Comment:
It's great to get test coverage here, but I'm concerned about testing a bug fix in core using an individual rule's test suite. It would be very easy for us to unintentionally lose coverage of this behavior and cause a regression. Can we test this in the config-validator or ESLint class's tests?

Reply:
I have added a config fixture for the reported issue. https://github.com/eslint/eslint/pull/13034/files#diff-3cbbbe2ccfd802293e0184c6ce75fea7R2. 

let me know if that's the one you are looking for.